02-microarray/dimension-reduction_microarray_01_pca.Rmd

```diff
@@ -0,0 +1,312 @@
+---
+title: "PCA Visualization - Microarray"
+author: "CCDL for ALSF"
+date: "`r format(Sys.time(), '%B %Y')`"
+output:   
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+# Purpose of this analysis
+
+<!--WHAT DOES THIS ANALYSIS DO AND WHEN WOULD YOU USE IT?-->
```

Comment:
I would remove this comment - folks will be able to see it when they download the Rmd file and it could be a point of confusion. https://github.com/AlexsLemonade/refinebio-examples/pull/183/files#r478728630

Reply:
Ahh gotcha, the instruction that references this will need to change in https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#how-to-use-the-templatermd as well.

Reply:
I figured it was in the instructions because I've seen it in two pull requests, but I don't understand the rationale.

Reply:
Gotcha, I am sure that it was a part of another conversation somewhere so I guess that's another question for when @cansavvy. 

However, a [brief conversation](https://github.com/AlexsLemonade/refinebio-examples/pull/165#discussion_r469551830) between @cansavvy and I on #165 suggests that at least part of the rationale here was: 
`That way when we review things we know where to expect added content.`